pkg/bazel/bazel.go

```diff
@@ -94,3 +95,31 @@ func (b *bazel) Flags() (map[string]*FlagInfo, error) {
 
 	return flags, nil
 }
+
+func (b *bazel) AQuery(query string) (*ActionGraphContainer, error) {
+	r, w := io.Pipe()
+	agc := &ActionGraphContainer{}
+
+	bazelErrs := make(chan error, 1)
+	defer close(bazelErrs)
+	go func() {
+		defer w.Close()
+		_, err := b.RunCommand([]string{"aquery", "--output=proto", query}, w)
+		bazelErrs <- err
+	}()
+
+	protoBytes, err := ioutil.ReadAll(r)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get Bazel aquery output: %w", err)
```

Comment:
The pattern for the error message should contain the current context. See my suggestion. This is so that we can migrate to Go 2 error handling with minimal effort. https://go.googlesource.com/proposal/+/master/design/go2draft-error-handling-overview.md

```suggestion
		return nil, fmt.Errorf("failed to run Bazel aquery: %w", err)
```